### ISSUE TITLE
bug([mat-input]): [mat-input dialog box is not fixed to the form field box]

### ISSUE THREAD

**Issue by estatheo** [angular/components#19079]
#### Reproduction

The bug is reproducible with an iphone with chrome browser (didn't test on safari)

Steps to reproduce:
1.  open https://material.angular.io/components/select/overview on version 9.2.0
2.  open one of the mat-select dropdowns and scroll the page
3. you will see the dropdown moving in the viewport
 
 
#### Expected Behavior

I would expected the dropdown dialog to be anchored to the form field box

#### Actual Behavior

The dropdown dialog is moving with the viewport

#### Environment

- Angular: 
- CDK/Material: 9,2.0
- Browser(s): Chrome on iOS
- Operating System (e.g. Windows, macOS, Ubuntu):  iOS

![bug](https://user-images.githubusercontent.com/57498775/79334859-f5eb4280-7f18-11ea-9ffc-ce7f3f4db3d0.jpg)

![bug](https://user-images.githubusercontent.com/22897412/79335439-f1735980-7f19-11ea-9554-a9d6c0f03057.jpg)

**Comment by Totati** [comment]
Are you and @tcaucorp the same person? :D

**Comment by estatheo** [comment]
@Totati yes, but I shouldn't have used that account, is it possible to delete #19078 ? apologize for the mess

**Comment by Totati** [comment]
It's closed, so won't bother anybody. I've just got a deja vu and got confused :D

**Comment by jelbourn** [comment]
Confirmed that `mat-select` isn't blocking scroll correctly on iOS

**Comment by rk-7** [comment]
Any updates on this issue? This issue exists even today after more than a year.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
